pkg/manage/cmd_config.go

```diff
@@ -0,0 +1,151 @@
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+)
+
+const configHelp = `Get or set config values
+
+This command gets or sets the config values for an organisation.
+
+Example:
+
+$ manage config voting
+disabled
+$ manage config set voting enabled
+$ manage config voting
+enabled
+`
+
+// CmdConfig initializes the config command.
+func CmdConfig(cfg *ClientConfig) *cobra.Command {
+	values := []string{
+		"voting",
+	}
+
+	cmd := &cobra.Command{
+		Use:       "config",
+		Short:     "Get or set config values.",
+		Long:      configHelp,
+		Args:      cobra.ExactValidArgs(1),
+		ValidArgs: values,
+
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+			defer cancel()
+
+			service, close, err := Dial(ctx, cfg.Address)
+			if err != nil {
+				return fmt.Errorf("connecting to gRPC server: %w", err)
+			}
+			defer close()
+
+			req := &proto.ConfigRequest{
+				Field: proto.ConfigRequest_Field(proto.ConfigRequest_Field_value[args[0]]),
+			}
+
+			resp, err := service.Config(ctx, req)
+			if err != nil {
+				return fmt.Errorf("get config value %s: %w", args[0], err)
+			}
+
+			fmt.Println(resp.Value)
+			return nil
+		},
+	}
+
+	cmd.AddCommand(&cobra.Command{
+		Use:   "set",
+		Short: "Set a value",
+		Args: func(cmd *cobra.Command, args []string) error {
+			if err := cobra.ExactArgs(2)(cmd, args); err != nil {
+				return err
+			}
+			for _, a := range values {
+				if args[0] == a {
+					return nil
+				}
+			}
+			return fmt.Errorf("invalid argument %s", args[0])
+		},
+		//Args:      cobra.ExactValidArgs(2),
+		//ValidArgs: values,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx, cancel := context.WithTimeout(context.Background(), cfg.Timeout)
+			defer cancel()
+
+			service, close, err := Dial(ctx, cfg.Address)
+			if err != nil {
+				return fmt.Errorf("connecting to gRPC server: %w", err)
+			}
+			defer close()
+
+			req := &proto.ConfigRequest{
+				Field:    proto.ConfigRequest_Field(proto.ConfigRequest_Field_value[args[0]]),
+				NewValue: args[1],
+			}
+
+			if _, err := service.Config(ctx, req); err != nil {
+				return fmt.Errorf("set config value %s to %s: %w", args[0], args[1], err)
+			}
+
+			return nil
+		},
+	})
+
+	return cmd
+}
+
+// Config gets or sets an organisation config value.
+func (s *Server) Config(ctx context.Context, in *proto.ConfigRequest) (*proto.ConfigResponse, error) {
+	var key string
+	switch in.Field {
+	case proto.ConfigRequest_VOTING:
+		key = "organisation/1/enable_electronic_voting"
+	default:
+		return nil, fmt.Errorf("Invalid request")
+	}
+
+	if in.NewValue == "" {
+		// Fetch value
+		waitForService(ctx, s.config.DatastoreReaderHost, s.config.DatastoreReaderPort)
+
+		addr := fmt.Sprintf("%s://%s:%s", s.config.DatastoreReaderProtocol, s.config.DatastoreReaderHost, s.config.DatastoreReaderPort)
```

Comment:
Maybe use package `url` instead of parsing a string.

Reply:
Can you give me an example?